tuist/tuist
# Add support for enabling markdown rendering in `Workspace.swift` for README files

* Added the option to include xcodesample.plist in workspace to render markdown readmes

Resolves https://github.com/tuist/tuist/issues/4356
Request for comments document (if applies):

### Short description 📝

> This PR has the purpose of making available the renderization of README.md in xcode,

### How to test the changes locally 🧐

> You can use the fixture workspace ios_app_with_custom_workspace_rendering_markdown,
 * Generate the workspace with tuist
 * Open the workspace and you should see a README.md rendered with Xcode
 * As you can see the parameter to render it is a boolean, that generates an empty plist file.

<img width="721" alt="image" src="https://user-images.githubusercontent.com/85997060/163593484-172cdb97-5890-4345-afc7-d6534d66d942.png">

I followed the steps marked to achieve this. Thanks to @kwridan and @danyf90 for the support.

### Checklist ✅
